src/hooks/useInstitution.ts

```
@@ -1,4 +1,60 @@
+  const [institution, setInstitution] = useState<InstitutionData | null>(null);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState<Error | null>(null);
+
+  useEffect(() => {
+    if (!config.institutionId) return;
+
+    let cancelled = false;
+    setLoading(true);
+
+    config.apiClient
+      .get<{ data: InstitutionData }>(`/institution/filter?filter=${config.institutionId}`)
```

ESLint

[error] Replace ``/institution/filter?filter=${config.institutionId}`` with `⏎········`/institution/filter?filter=${config.institutionId}`⏎······`

(prettier/prettier)